skins/common/shared.css

```diff
@@ -1047,5 +1047,5 @@ table.floatleft {
 /* For developpers */
 .xdebug-error {
 	position: absolute;
-	z-index: 99;
+	z-index: 99999999; /* Wikia change */
```

Comment:
Given the contents of many debug messages, this change implies our devboxes are now adults only (18+).